network_runner/tests/unit/test_api.py

```diff
@@ -96,6 +110,19 @@ def test_remove_access_port_raises(self, m_run_task):
                           self.net_runr.delete_port,
                           self.testhost, self.testport)
 
+    def test_assign_access_port_w_kwarg(self, m_run_task):
+        self.net_runr.conf_access_port(self.testhost,
+                                       self.testport,
+                                       self.testvlan,
+                                       mykwarg='noval')
+        m_run_task.assert_called_once()
```

Comment:
These should all be assert_called_once_with() to check mykwarg gets through to the task

Reply:
Agreed, I started there and couldn't figure out how to match all the parameters I'll revisit as I think things have changed such it warrants a revisit. worse case I'll add a TODO so we can revisit.